flow_control/tests/test_flowcontrol.py

```diff
@@ -98,29 +99,38 @@ def test_get_location_string(self):
         It should return the problem location string given its Id
         """
 
+        course_strings = [
+            'course',
+            'course-v1',
+            'Course',
+            'course+course',
```

Comment:
Do you think this should be a valid string? or are you expecting the tests to fail when they reach this value?

Reply:
Is just for fail in case. @felipemontoya